Framework/CurveFitting/test/CompositeFunctionTest.h

```diff
@@ -147,6 +147,31 @@ class CompositeFunctionTest : public CxxTest::TestSuite {
         "curvefitting.peakRadius", m_preSetupPeakRadius);
   }
 
+  void test_has_parameter() {
+    auto mfun = boost::make_shared<CompositeFunction>();
+    auto g = boost::make_shared<CurveFittingGauss>();
+    auto g1 = boost::make_shared<CurveFittingGauss>();
+    auto g2 = boost::make_shared<CurveFittingGauss>();
+    auto bk = boost::make_shared<CurveFittingLinear>();
+    auto c = boost::make_shared<CompositeFunction>();
```

Comment:
A minor thing, but could we rename these to make them a bit clearer? e.g. gauss instead of g